src/com/sun/ts/tests/webservices13/servlet/WSRespBindAndAddressingTestUsingAnnotations/custom-client.xml

```diff
@@ -17,7 +17,7 @@
 
 -->
 
-<bindings wsdlLocation="EchoService.wsdl" xmlns="http://java.sun.com/xml/ns/jaxws">
+<bindings wsdlLocation="EchoService.wsdl" xmlns="https://jakarta.ee/xml/ns/jaxws">
```

Comment:
line no. 26 can be updated to jakartaee ns

Reply:
fixed